### ISSUE TITLE
Add some tests for the provided examples

### ISSUE THREAD

**Issue by giacomocavalieri** [giacomocavalieri/glam#5]
I'd like to add some tests for the [examples](https://github.com/giacomocavalieri/glam/tree/main/src/examples) that are shown in the library.
This would be beneficial in two ways:

- act as some sort of "integration testing" to make sure that future changes do not break the pretty printing algorithm
- make sure I do not show buggy code in the tutorials :P

### _Why is this a good first issue and how can you help?_
I feel like this shouldn't be too complicated, especially if one has already walked through the package's tutorials.
Basically, what one would need to do is:

- read the examples' code
- make the `x_to_doc` functions to test public
- create a new module in the `tests` folder to add the tests for each pretty printer
- write some tests that test different data structures at different line widths
  - you can use the tutorials' examples as a reference for how the pretty printed documents should look

**Comment by giacomocavalieri** [comment]
I've completed this issue using [Birdie](https://github.com/giacomocavalieri/birdie)!
